front/src/modules/ui/field/hooks/useResetField.ts

```diff
@@ -0,0 +1,73 @@
+import { useContext } from 'react';
+
+import { ColumnIndexContext } from '@/ui/data-table/contexts/ColumnIndexContext';
+
+import { FieldContext } from '../contexts/FieldContext';
+import { isFieldBoolean } from '../types/guards/isFieldBoolean';
+import { isFieldChip } from '../types/guards/isFieldChip';
+import { isFieldDate } from '../types/guards/isFieldDate';
+import { isFieldDoubleText } from '../types/guards/isFieldDoubleText';
+import { isFieldDoubleTextChip } from '../types/guards/isFieldDoubleTextChip';
+import { isFieldEmail } from '../types/guards/isFieldEmail';
+import { isFieldMoney } from '../types/guards/isFieldMoney';
+import { isFieldNumber } from '../types/guards/isFieldNumber';
+import { isFieldPhone } from '../types/guards/isFieldPhone';
+import { isFieldProbability } from '../types/guards/isFieldProbability';
+import { isFieldRelation } from '../types/guards/isFieldRelation';
+import { isFieldText } from '../types/guards/isFieldText';
+import { isFieldURL } from '../types/guards/isFieldURL';
+
+import { useIsFieldEmpty } from './useIsFieldEmpty';
+import { usePersistField } from './usePersistField';
+
+export const useResetField = () => {
```

Comment:
This should be scoped to the table, so to be moved in data table folder and renamed like `useResetTableCell`